.eslintignore

```diff
@@ -15,3 +15,4 @@ coverage
 lint-staged.config.mjs
 vite.config.js
 packages/*/lib
+.storybook
```

Comment:
I was getting warnings about the files in this directory being ignored. I don't know if there is a better way to handle this.